index.js

```diff
@@ -59,6 +59,23 @@ module.exports = {
     // Prefer named export
     'import/prefer-default-export': 'off',
     'import/no-default-export': 'error',
+
+    // Order import with line separation between groups
+    'import/order': [
+      'error',
+      {
+        // Add module import (@Module) after external import
+        pathGroups: [
+          {
+            pattern: '@*',
```

Comment:
Be careful here with this pattern.
Some package like for example `@nestjs/common` use '@' at begining which is not a locale directory.

Maybe target just only local file if possbile of new line required.(also regular import)